scripts/utils/price-utils.js

```diff
@@ -134,9 +137,48 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
     return true
   }
 
+  /**
+   * Computes the amount of output token units from their price and the amount of input token units
+   * Note that the price is expressed in terms of tokens, while the amounts are in terms of token units
+   * @param {number} price amount of output token in exchange for one input token
+   * @param {BN} targetTokenAmount amount of token units that are exchanged at price
+   * @param {integer} targetTokenDecimals number of decimals of the input token
+   * @param {integer} stableTokenDecimals number of decimals of the output token
+   * @return {BN} amount of output token units obtained
+   */
+  const getOutputAmountFromPrice = function(price, targetTokenAmount, targetTokenDecimals, stableTokenDecimals) {
```

Comment:
Love the documentation! This function looks a lot like an already existing (undocumented function) `computeBuyAndSellAmountsFromPrice`. Do you think this would be an appropriate PR to include some doc strings on that as well?